src/setupToastify.ts

```diff
@@ -1,8 +1,13 @@
-import 'react-toastify/dist/ReactToastify.min.css'
+// import 'react-toastify/dist/ReactToastify.min.css'
+import 'assets/css/ReactToastifyCustom.scss'
```

Comment:
At this point just add to `GlobalStyles` in `src/styles/global`, then will be no need to bring in sass
```
const GlobalStyles = createGlobalStyle`
${ReactToastify}
```
or something